### PR TITLE
sep31: /info endpoint

### PR DESCRIPTION
**Pull request by lijamie98**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Implement the Sep31 `/info` endpoint.

### CODE REVIEW

**Review by marcelosalloum — APPROVED** [review]
LGTM! There are just a few suggestions below.

File: core/src/main/java/org/stellar/anchor/util/PropertyUtil.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+public class PropertyUtil {
+  @SuppressWarnings("unchecked")
+  public static void set(Object instance, String path, Object value)
+      throws ReflectiveOperationException {
+    Target location = findTarget(instance, path, true);
+    // Now we have reached the final target
+    if (location.instance instanceof Map) {
+      Map<String, Object> map = (Map<String, Object>) location.instance;
+      map.put(location.field, value);
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply core/src/main/java/org/stellar/anchor/util/PropertyUtil.java:17]: Thanks for the good information. I will modify accordingly.

File: core/src/main/java/org/stellar/anchor/util/PropertyUtil.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  @SuppressWarnings("unchecked")
+  public static Optional<Object> get(Object bean, String path) {
+    try {
+      Target target = findTarget(bean, path, false);
+      if (target.instance == null) {
+        return Optional.empty();
+      }
+      if (target.instance instanceof Map) {
+        Map<String, Object> map = (Map<String, Object>) target.instance;
+        return Optional.ofNullable(map.get(target.field));
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply core/src/main/java/org/stellar/anchor/util/PropertyUtil.java:32]: Thanks for the good information. I will modify accordingly.

File: core/src/main/java/org/stellar/anchor/util/PropertyUtil.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  @SuppressWarnings("unchecked")
+  static Target findTarget(Object target, String path, boolean create)
+      throws ReflectiveOperationException {
+    StringTokenizer st = new StringTokenizer(path, ".");
+    String name;
+    for (name = st.nextToken(); st.hasMoreTokens(); name = st.nextToken()) {
+      if (target instanceof Map) {
+        Map<String, Object> map = (Map<String, Object>) target;
+        target = map.get(name);
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply core/src/main/java/org/stellar/anchor/util/PropertyUtil.java:49]: Yes, it works.

File: platform/example.anchor-config.yaml [hunk 4]
```diff
@@ -54,6 +54,10 @@ app-config:
     enabled: true
     customerIntegrationEndpoint: http://localhost:8081
 
+  # sep-31
+  sep31:
+    enabled: true
+
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply platform/example.anchor-config.yaml:60]: Yes, thanks for catching this.

File: platform/src/test/resources/test_assets.json [hunk 5]
```diff
@@ -3,72 +3,82 @@
...
-        "min_amount": 1,
+        "min_amount": 0,
         "max_amount": 10000
       },
-      "sep38": {
-        "exchangeable_assets": [
-          "stellar:JPYC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5",
-          "iso4217:USD"
-        ]
-      },
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply platform/src/test/resources/test_assets.json:33]: This is git diff display problem. They are not removed.
https://github.com/stellar/java-stellar-anchor-sdk/blob/develop-sep31/platform/src/test/resources/test_assets.json#L74

**Pull request merged**